css/style.css

```diff
@@ -1,19 +1,506 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* Header */
+.header-section {
+  display: flex;
+  flex-flow: row nowrap;
+  width: 100%;
+  padding: 1rem 0 1rem 0;
+}
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+/* Navigation */
+.company-logo {
+  padding-left: 0.8rem;
+  width: 2.3rem;
+  height: 2.3rem;
+}
+
+.hamburger-icon {
+  padding: 0.8rem 0.8rem 0 0;
+  width: 1.5rem;
+  height: 1.1rem;
+}
+
+.navbar {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-end;
+  width: 100%;
+}
+
+.navbar-hamburger-Menu {
+  display: flex;
+  flex-flow: column nowrap;
```

Comment:
```suggestion
```
This button element has one child which is an image. Creating a flex column doesn't do anything here, so we should remove it